ethereum-types/Cargo.toml

```diff
@@ -22,7 +22,7 @@ scale-info = { version = "1.0", features = ["derive"], default-features = false,
 serde_json = "1.0.41"
 
 [features]
-default = ["std", "rlp", "serialize"]
+default = ["std", "ethbloom", "rlp", "serialize"]
 std = ["uint-crate/std", "fixed-hash/std", "ethbloom/std", "primitive-types/std"]
```

Comment:
Do you want to move `ethbloom/std` to default features as well in order for `std` not to pull ethbloom?